tools/cli/commands.js

```diff
@@ -289,10 +290,16 @@ function doRunCommand(options) {
   const { parsedServerUrl, parsedMobileServerUrl } =
     parseServerOptionsForRunCommand(options, runTargets);
 
+  var includePackages = [];
+  if (options['extra-packages']) {
+    includePackages = options['extra-packages'].split(',');
```

Comment:
Agree with this, but I think I'd have a strong preference to:

```js
includePackages = options['extra-packages'].split(/\s+,\s+/)
```
...which I _believe_ would be more performant.